diff -rupN keepassx-0.4.3.old/src/lib/random.cpp keepassx-0.4.3/src/lib/random.cpp
--- keepassx-0.4.3.old/src/lib/random.cpp	2009-06-28 16:22:31.000000000 +0200
+++ keepassx-0.4.3/src/lib/random.cpp	2012-10-09 16:53:35.000000000 +0200
@@ -18,9 +18,10 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+#include <sys/types.h>
+#include <unistd.h>
 #include "random.h"
 
-
 #if defined(Q_WS_X11) || defined(Q_WS_MAC)
 	#include <QFile>
 #elif defined(Q_WS_WIN)
